src/resolution.js

```diff
@@ -327,13 +352,13 @@ var ri = module.exports = {
 };
 
 ri.config = util.clone(configDefaults);
-ri.init();
+ri.init(document.body);
```

Comment:
This should use `{measurementNode: document.body}` as below, right?

Reply:
Also noticed (in strawman, at least) that this code runs before `body` has been created so both the current code and my suggested replacement are equivalent and likely wrong. Do we need to wrap this in `ready()` to wait for a valid node reference?

Reply:
Yes, you're probably right @ryanjduffy. It will need to run after the dom-ready event has fired, or at least the `body` tag is present.